lib/workers/repository/common.ts

```diff
@@ -10,9 +11,24 @@ export function extractRepoProblems(
         (problem) =>
           problem.repository === repository && !problem.artifactErrors,
       )
-      .map(
-        (problem) =>
-          `${nameFromLevel[problem.level].toUpperCase()}: ${problem.msg}`,
-      ),
+      .map((problem) => `${formatProblemLevel(problem.level)}: ${problem.msg}`),
   );
 }
+
+type EmojiLogLevelMapping = Record<number, string>;
+
+const logLevelEmojis: EmojiLogLevelMapping = {
+  [TRACE]: ':microscope:',
+  [DEBUG]: ':mag:',
+  [INFO]: ':information_source:',
+  [WARN]: ':warning:',
+  [ERROR]: ':x:',
+  [FATAL]: ':skull:',
+};
+
+export function formatProblemLevel(level: number): string {
+  const name = nameFromLevel[level].toUpperCase();
+  const unicode = logLevelEmojis[level];
+
+  return unicode ? `${emojify(unicode)} ${name}` : name;
```

Comment:
```suggestion
  const emojiName = logLevelEmojis[level];

  return emojiName ? `${emojify(emojiName)} ${name}` : name;
```

Or something similar, because it's now not actually the `unicode`